specs/bellatrix/fork-choice.md

```diff
@@ -74,7 +74,7 @@ As per EIP-3675, before a post-transition block is finalized, `notify_forkchoice
 ##### `safe_block_hash`
 
 The `safe_block_hash` parameter MUST be set to return value of
-[`get_safe_execution_payload_hash(store: Store)`](../../fork_choice/safe-block.md#get_safe_execution_payload_hash) function.
+[`get_safe_execution_payload_hash(store: Store)`](../../fork_choice/confirmation-rule.md#get_safe_execution_payload_hash) function.
```

Comment:
I would make it into a separate PR after the general confirmation rule algo is merged

Reply:
I am Ok with this.
@adiasg @djrtwo , what do you think?

Reply:
I'm fine with decoupling it from this PR 👍

Reply:
In fact, if it's *not* a separate feature, we can't easily merge this PR due to compatibility.

Reply:
Done.